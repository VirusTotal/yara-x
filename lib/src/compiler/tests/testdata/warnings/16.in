rule test {
  strings: 
    $a = "foo"
  condition: 
    for any of them: ( 1 )
}