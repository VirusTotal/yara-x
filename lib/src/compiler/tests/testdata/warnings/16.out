warning: non-boolean expression used as boolean
 --> line:5:24
  |
5 |     for any of them: ( 1 )
  |                        - this expression is `integer` but is being used as `bool`
  |
  = note: non-zero integers are considered `true`, while zero is `false`